app/services/ccms/applicant_search_requestor.rb

```diff
@@ -1,5 +1,10 @@
 module CCMS
-  class ClientSearchRequestor < BaseRequestor
+  class ApplicantSearchRequestor < BaseRequestor
+    def initialize(applicant)
+      @applicant = applicant
+      super()
```

Comment:
`super`. There is no need for the `()` in ruby.

Reply:
I think in this instance we do need the (). Unlike `initialize` in this class, the `initialize` method in `BaseRequestor` doesn't take any arguments. Calling `super` attempts to pass `applicant` through to the parent, which causes an error. `super()` doesn't pass the argument.